.github/workflows/workflow.yaml

```diff
@@ -143,65 +143,136 @@ jobs:
+      - run: ./setup.sh -t -p master                 | grep 'flutter-linux-master-any-x64-master'
+        shell: bash
+      - run: ./setup.sh -t -p -n 5b12b74 master      | grep 'flutter-linux-master-5b12b74-x64-master'
+        shell: bash
+      - run: ./setup.sh -t -p -n 3.12.0-12.0.pre master     | grep 'flutter-linux-master-3.12.0-12.0.pre-x64-master'
         shell: bash
-      - run: ./setup.sh -t -p -n 1 -a x64 stable     | grep '1.22.6'
+      - run: ./setup.sh -t -p -n stable master       | grep 'flutter-linux-master-stable-x64-master'
         shell: bash
-      - run: ./setup.sh -t -p -n 0 -a x64 any        | grep 'beta'
+      - run: ./setup.sh -t -p -n 2 any               | grep 'flutter-linux-beta-2.13.0-0.4.pre-x64-25caf1461b8f643092a9f6f5b224453b5c057d10'
         shell: bash
-      - run: ./setup.sh -t -p -n 0 -a x64 any        | grep '0.11.13'
+      - run: ./setup.sh -t -p -n 1 any               | grep 'flutter-linux-beta-1.26.0-17.8.pre-x64-044f2cf5607a26f8818dab0f766400e85c52bdff'
         shell: bash
-      - run: ./setup.sh -t -p -a x64                 | grep 'flutter-macos-stable-3.7.7-x64-2ad6cd72c040113b47ee9055e722606a490ef0da'
+      - run: ./setup.sh -t -p -n 0 any               | grep 'flutter-linux-beta-0.11.13-x64-58c8489fcdb4e4ef6c010117584c9b23d15221aa'
         shell: bash
-      - run: ./setup.sh -t -p -a x64 stable          | grep 'flutter-macos-stable-3.7.7-x64-2ad6cd72c040113b47ee9055e722606a490ef0da'
+      - run: ./setup.sh -t -p                        | grep '/opt/hostedtoolcache/flutter/stable-3.7.7-x64'
         shell: bash
-      - run: ./setup.sh -t -p -a x64 beta            | grep 'flutter-macos-beta-3.9.0-0.1.pre-x64-f732038a8cf4562ce38a1d7debb30209ae3da896'
+      - run: ./setup.sh -t -p stable                 | grep '/opt/hostedtoolcache/flutter/stable-3.7.7-x64'
         shell: bash
-      - run: ./setup.sh -t -p -a x64 dev             | grep 'flutter-macos-dev-2.11.0-0.1.pre-x64-b101bfe32f634566e7cb2791a9efe19cf8828b15'
+      - run: ./setup.sh -t -p beta                   | grep '/opt/hostedtoolcache/flutter/beta-3.9.0-0.1.pre-x64'
+        shell: bash
+      - run: ./setup.sh -t -p dev                    | grep '/opt/hostedtoolcache/flutter/dev-2.13.0-0.1.pre-x64'
+        shell: bash
+      - run: ./setup.sh -t -p master                 | grep '/opt/hostedtoolcache/flutter/master-any-x64'
+        shell: bash
+      - run: ./setup.sh -t -p -k 'custom-:channel:-:version:-:hash:'       | grep 'custom-stable-3.7.7-2ad6cd72c040113b47ee9055e722606a490ef0da'
+        shell: bash
+      - run: ./setup.sh -t -p -k 'custom-:channel:-:version:-:sha256:'     | grep 'custom-stable-3.7.7-cdd49597e88c35e5de5184e9acff433dfbb3c075b74ef7b27790af7f57ce2420'
+        shell: bash
+      - run: ./setup.sh -t -p -c '/tmp/flutter/:channel:-:version:-:hash:' | grep '/tmp/flutter/stable-3.7.7-2ad6cd72c040113b47ee9055e722606a490ef0da'
+        shell: bash
+
+
+  test_print_output_arm64:
+    runs-on: macos-latest
```

Comment:
Does the `runs-on` affect if the tests will be successful or not?

Reply:
Currently, arm64 is only available for macos.
Also, if macos-latest ever changes from arm64, you will notice the change in this setting. (I would like to believe that this is not the case.)

Reply:
gotcha - thank you!